src/mgr/Mgr.cc

```diff
@@ -496,6 +502,35 @@ void Mgr::handle_service_map(MServiceMap *m)
   server.got_service_map();
 }
 
+void Mgr::handle_monmap(MMonMap *m)
+{
+  assert(lock.is_locked_by_me());
+  std::set<std::string> names_exist;
+
+  for (unsigned int i = 0; i < monmap.size(); i++) {
```

Comment:
This won't build (`monmap` is an undeclared variable).

Reply:
Thanks for your review @tserong . The build error has been fixed now.